index.html

```diff
@@ -376,113 +396,203 @@ <h4>Characters</h4>
-          <p class="issue" data-number="20"></p>
+          <p>A <a>Text</a> object is represented with a <code>p</code> element with the following constraints:
+            <ul>
+              <li>The text content of the <code>p</code> is the <a>Text</a> of the <a>Event</a>. It MAY also use TTML markup to represent styles.
+                <div class="note">The text content of the paragraph may contain TTML elements such as <code>span</code> or TTML attributes such as <code>tts:ruby</code> used to alter the layout or styling of each paragraph</div>
+              </li>
+              <li>The <code>p</code> element MUST have an <code>xml:lang</code> attribute corresponding to the language of the <a>Text</a> object.</li>
+          <pre class="example">
+&lt;div xml:id=&quot;event_0130df43-5443-4206-b1d9-d331e42b5a86&quot;
+     begin=&quot;9663f&quot; end=&quot;9682f&quot; 
+     style=&quot;style_bb89e9c6-4453-4402-a376-5a496cfb9839&quot;
+     ttm:agent=&quot;character_bb89e9c6-4453-4402-a376-5a496cfb9839&quot;&gt;
+  &lt;p xml:lang=&quot;pt-BR&quot; &gt;Voc&#xea; vai ter.&lt;/p&gt;
+  &lt;p xml:lang=&quot;fr&quot; ttm:role=&quot;x-context&quot;&gt;Bah, il arrive.&lt;/p&gt;
+&lt;/div&gt;
+          </pre>
         </section>
 
         <section>
           <h4>On Screen</h4>
-          <p>Each <a>Event</a> MAY have a <dfn>On Screen</dfn> property, which is an annotation indicating the position of the <a>character</a> during the <a>Event</a>:
+          <p>The <dfn>On Screen</dfn> property is an annotation indicating the position in the scene of the <a>character</a> during the <a>Event</a>:
             <ul>
               <li>ON - the <a>character</a> is on screen for the entire duration</li>
               <li>OFF - the <a>character</a> is off screen for the entire duration</li>
               <li>ON_OFF - the <a>character</a> starts on screen, but goes off screen at some point</li>
               <li>OFF_ON - the <a>character</a> starts off screen, but goes on screen at some point</li>
-            </ul>as a human-readable description of the <a>Event</a>. 
+            </ul>
+          </p>
+          <p>The <a>On Screen</a> property is represented as a <code>metadata</code> element with the following constraints:
+            <ul>
+              <li>The following attribute corresponding to the <a>On Screen</a> <a>Event</a> property may be present:
+                <div class="exampleInner">
+                  <pre class="nohighlight">
+daptm:onScreen
+  : "ON" 
+  | "OFF"
+  | "ON_OFF"
+  | "OFF_ON"
+                  </pre>
+                </div>
+                <p class="issue" data-number="34"></p>
+              </li>
+              <li>A <code>ttm:desc</code> element may be present representing the <a>Event Description</a>.</li>
```

Comment:
may be present on what element?

Reply:
ttm:desc is an element. It was meant to be in the under the `p` representing the event.

Reply:
Oh, right, that needs clarifying.

Reply:
#64